novawallet/Modules/NetworkManagement/NetworksList/NetworksListViewController.swift

```diff
@@ -104,21 +111,29 @@ extension NetworksListViewController: UITableViewDataSource {
             )
         }
     }
+
+    func tableView(_: UITableView, viewForFooterInSection _: Int) -> UIView? {
+        UIView()
```

Comment:
What we are trying to fix with that approach?

Reply:
It's just to create a space between sections according to the design, so we set a footer of the desired height for that.